.eslintrc.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
       {
         "devDependencies": [
           "config/*.js",
-          "**/*.test.jsx"
+          "**/*.test.jsx",
```

Comment:
Ugh, there's probably some really simple regex that combines these.

![alt-text](https://media.giphy.com/media/xkfGEJnTdgYGQ/giphy.gif)